### DISCUSSION TITLE
Changing component color palette

### DISCUSSION THREAD

**Discussion by dhrblav** [molstar/molstar#484] [Q&A] [answered]
Hi,
I can easily change components coloring using updateRepresentationsTheme(), passing eg. { color: 'hydrophobicity' }.
For those coloring schemes that support color lists (eg. hydrophobicity), I would now want to programmatically change the color list to one of the other built-in color lists or to a custom one. I have tried to pass in UpdateThemeParams various palette-related parameters such as:
`colorParams: {
   palette: {
      filter: 'nearest',
      colors: [0xBF2222, 0xFFFFFF, 0x3361E1]
   }
}`
But nothing I tried had any effect: the color list does not change.
So I must be missing something, possibly trivial, and I am now asking for help...
Thank you.

**Comment by dsehnal** [comment, marked as answer]
I don't think we ever call this argument `palette`. Usually, this is called ``list`` or ``colorList``. So it should be ``colorParams: {  list: { ...} }`` and not what you are using.
